t/apicast-policy-http-proxy.t

```diff
@@ -202,10 +202,8 @@ ETag: foobar
 --- error_code: 200
 --- error_log env
 proxy request: CONNECT test-upstream.lvh.me:$TEST_NGINX_RANDOM_PORT HTTP/1.1
---- user_files fixture=tls.pl eval
---- error_log env
```

Comment:
there cannot be two `--- error_log` directives. Only one of them is being applied.